ignite3/src/main/java/site/ycsb/db/ignite3/IgniteJdbcClient.java

```diff
@@ -48,34 +48,29 @@ public void init() throws DBException {
     }
   }
 
+  /** {@inheritDoc} */
   @Override
   public Status read(String table, String key, Set<String> fields, Map<String, ByteIterator> result) {
     try {
-      String qry = prepareReadStatement(table);
+      PreparedStatement stmt = prepareReadStatement(CONN.get());
```

Comment:
We need to close `PreparedStatement` instance somewhere. If it's supposed to be reused among various invokations, the best place to do it seems to be in `cleanup()` method. Seems like an additional argument for moving these two fields into `IgniteJdbcClient` class.